### ISSUE TITLE
Script "make-local" is missing

### ISSUE THREAD

**Issue by rain0r** [glowing-bear/glowing-bear#1184]
The FAQ says:

> run npm run make-local to download the files and apply a patch to use them instead of the CDN.

But there is no script `make-local` in `package.json` 

```
$ npm run make-local
npm ERR! missing script: make-local
```

**Comment by lorenzhs** [comment]
Yeah that bit of the FAQ is outdated since we moved to using webpack. I think we should just add a mode where the remaining dependencies are also added to the webpack config and then instead of `make-local` it would just be `build`. For the moment, `npm run build` gets you most of the way there.

**Comment by rain0r** [comment]
Thanks, `npm run build` is working.

Just one more thing: Is it possible to patch the includes to be local? Like, not from `google.com`?

**Comment by lorenzhs** [comment]
Google? They should be loaded from `cnjs.cloudflare.com`, not google.

It's not possible with the current script sadly, but we should add such a mode again. With browsers increasingly not reusing cached resources between different sites, the utility of a CDN has become very very low. Likely, we should make that the default mode. If you want to have a go, have a look at `src/main.js` and `webpack.config.js`.

**Comment by rain0r** [comment]
> Google? They should be loaded from `cnjs.cloudflare.com`, not google.

Yes, my mistake, you're right.

> It's not possible with the current script sadly, but we should add such a mode again. With browsers increasingly not reusing cached resources between different sites, the utility of a CDN has become very very low. Likely, we should make that the default mode. If you want to have a go, have a look at `src/main.js` and `webpack.config.js`.

I created https://github.com/glowing-bear/glowing-bear/pull/1186 for that.

But I'm no webpack-expert at all so there might be a thing or two to optimize.
